lib/facter/pe_status_check.rb

```diff
@@ -219,6 +219,19 @@
     end
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if current connections to Postgres exceeds 90% of max defined
+
+    maximum = PEStatusCheck.max_connections.to_f
+    current = PEStatusCheck.cur_connections.to_f
+    percent_used = (current / maximum) * 100
+    { S0029: percent_used <= 90 }
+  rescue ZeroDivisionError
+    Facter.debug('pe_status_check.S0029 failed to get max_connections')
```

Comment:
Could we change this to a `Facter.warn`?  A `warn` level message will show up when running `puppet agent -t` or `facter` interactively, while `debug` always goes to the logs.  We have an example [here](https://github.com/puppetlabs/puppetlabs-pe_status_check/blob/7f5ae6693da967f5ba7b386e195a0064a7988b75/lib/shared/pe_status_check.rb#L109-L112) where we use `warn` to display the error and `debug` to log the stack trace.  I don't think we need the backtrace here since we're capturing a specific exception.

Reply:
done